go.mod

```diff
@@ -9,7 +9,7 @@ require (
 	github.com/codahale/hdrhistogram v0.0.0-20160425231609-f8ad88b59a58 // indirect
 	github.com/containerd/cgroups v0.0.0-20181219155423-39b18af02c41 // indirect
 	github.com/containerd/console v0.0.0-20180822173158-c12b1e7919c1
-	github.com/containerd/containerd v1.3.0-0.20190212172151-f5b0fa220df8
+	github.com/containerd/containerd v1.3.0-0.20190321141026-ceba56893a76
```

Comment:
@AkihiroSuda btw, how did you get this date `20190321141026` ?

Reply:
`go get github.com/containerd/containerd@master` gave `github.com/containerd/containerd v1.2.1-0.20190321141026-ceba56893a76`

Reply:
Do we need to document that somewhere? (or have a make-target to make it easier?)